lib/config-generator.js

```diff
@@ -86,7 +100,7 @@ class ConfigGenerator {
         }
 
         if (this.webpackConfig.useDevServer()) {
```

Comment:
What about this, since we alraedy check `this.webpackConfig.useDevServer()` before?

```suggestion
        if (null !== devServerConfig) {
```